rest_api/schema.py

```diff
@@ -41,4 +43,5 @@ class QueryResponse(BaseModel):
     query: str
     answers: List[AnswerSerialized]
     documents: Optional[List[DocumentSerialized]]
+    debug: Optional[Dict] = Field(None, alias="_debug")
```

Comment:
I'd say let's not alias it: `debug` is usually the boolean value, `_debug` is the data dictionary (at least, that's how it works in the Pipeline).

On a general note, probably it would be clearer to have the two called `debug` and `debug_data` or something similar, to distinguish them better. The prefix underscore seems to imply `_debug` is a private attribute, which is not the case... But if we decide to do this I would update the Pipeline code too, so that they match.

Reply:
The problem here is that we can't return the key `_debug` as anything that starts with a `_` is considered a private attribute in fast API and is removed from the response to the client. Took me some time to figure this out and find this suggested workaround somewhere.

Reply:
Didn't know that! :+1: